test/Microsoft.ML.Tests/Scenarios/TensorflowTests.cs

```diff
@@ -37,7 +37,7 @@ public void TensorFlowTransforCifarEndToEndTest()
             var pipeEstimator = new ImageLoadingEstimator(mlContext, imageFolder, ("ImageReal", "ImagePath"))
                     .Append(new ImageResizingEstimator(mlContext, "ImageCropped", imageHeight, imageWidth, "ImageReal"))
                     .Append(new ImagePixelExtractingEstimator(mlContext, "Input", "ImageCropped", interleave: true))
-                    .Append(new TensorFlowEstimator(mlContext, new[] { "Output" }, new[] { "Input" }, model_location))
+                    .Append(mlContext.Transforms.ScoreTensorFlowModel(model_location, new string[] { "Output" }, new[] { "Input" }))
```

Comment:
>new string[] { "Output" }, new[] { "Input" }) [](start = 86, length = 45)

do you want to use the other extension, with string signatures for input and output, rather than the arrays?

Reply:
Sure!

---
In reply to: [254387227](https://github.com/dotnet/machinelearning/pull/2367#discussion_r254387227) [](ancestors = 254387227)